src/Common/src/CoreLib/System/ComponentModel/DefaultValueAttribute.cs

```diff
@@ -36,7 +36,24 @@ public DefaultValueAttribute(Type type, string value)
             // load an otherwise normal class.
             try
             {
-                if (type.IsSubclassOf(typeof(Enum)))
+#if MONO
```

Comment:
Please change this to `__MonoCS__` which is mcs specific define

Reply:
done